frontend/lib/norent/data/faqs-content.tsx

```diff
@@ -0,0 +1,118 @@
+import React from "react";
+
+export type FaqCategory =
+  | "Letter Builder"
+  | "Tenant Rights"
+  | "Connecting With Others"
+  | "After Sending Your Letter";
+
+export type Faq = {
+  question: string;
+  category: FaqCategory;
+  priority: number;
+  answerPreview: React.ReactNode;
+  answerFull: React.ReactNode;
+};
+
```

Comment:
While I know we will eventually be storing this data somewhere else, I added these type definitions to solidify our "data structure" here so that I could build out the logic of the FAQs page in preparation for having the data ready.